python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -268,6 +272,7 @@ def empty(cls, target_subset: AssetGraphSubset) -> "AssetBackfillData":
             materialized_subset=AssetGraphSubset(asset_graph),
             failed_and_downstream_subset=AssetGraphSubset(asset_graph),
             latest_storage_id=None,
+            evaluation_time=evaluation_time,
```

Comment:
maybe this field could be called something like "backfill_start_time" or "backfill_time", as "evaluation_time" is a bit confusing.

Reply:
good call--I'll switch to `backfill_start_time`